tfhe/src/core_crypto/entities/compressed_modulus_switched_lwe_ciphertext.rs

```diff
@@ -250,6 +259,25 @@ impl<Scalar: UnsignedTorus> CompressedModulusSwitchedLweCiphertext<Scalar> {
     }
 }
 
+impl<Scalar: UnsignedTorus> ParameterSetConformant
```

Comment:
does it need to be UnsignedTorus ? Maybe UnsignedInteger is enough ?

Reply:
Changed to UnsignedInteger